modules/ROOT/pages/document-types.adoc

```diff
@@ -0,0 +1,75 @@
+= Document Types
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Configuring a document type for a partner means configuring settings specific to that document type.
```

Comment:
What is a document type? Where does it fit/how is it related to a message flow? Transaction? message? etc.

Reply:
I thought users understood what a document type was in a flow. If not, then need to add the definition here.